### PR TITLE
Update docs after beta4

### PR DESCRIPTION
**Pull request by sashayakovtseva**

### CODE REVIEW

File: docs/constraints.rst [hunk 1]
```diff
@@ -20,9 +20,7 @@ for additional details.
 Using image from private registry
 ---------------------------------
 
-Unfortunately, it is not possible to use private Docker registry of Sylabs library when working with
-Singularity-CRI. This is an object to change soon, we are working with Kubernetes maintainers on
+Unfortunately, it is not possible to use on-prem Sylabs library or pull private SIF images when working with
+Singularity-CRI. This is a subject to change soon, we are working with Kubernetes maintainers on
```
Comment (sushma-4) [inline docs/constraints.rst]:
```suggestion
Singularity-CRI. However, this is expected to change soon as we work with Kubernetes maintainers on
```

**Review by sushma-4 — APPROVED** [review]
Looks Good! Just made one suggestion

**Comment by sashayakovtseva** [comment]
@sushma-4 can you upload the changes to sylabs.io/docs?

**Comment by sushma-4** [comment]
@devprincess still handles that, I might mess up something here. I just asked her to send these changes.

**Comment by jchavezlavalle** [comment]
@sashayakovtseva @sushma-4  See PR:  https://github.com/sylabs/corp-website/pull/41 , please for further deployments do refer to other members of the team that maintain the corp-website repo.

**Pull request merged**